app/views/stats/index.html.erb

```diff
@@ -24,12 +24,18 @@
 <div class="wrapper">
   <dl class="statistics meta group" role="complementary">
     <% [:user_subscriptions, :kudos, :comments, :bookmarks, :subscriptions, :word_count].each do |stat| %>
-      <dt><%= (stat.to_s.titleize == "Comments") ? ts("Comment Threads") : stat.to_s.titleize %></dt>
-      <dd><%= @totals[stat] %></dd>
+      <% if stat == :word_count
+          html_class = 'words'
+        else
+          html_class = stat.to_s.humanize.downcase
+        end
+      %>
+      <dt class="<%= html_class %>"><%= (stat == :comments) ? ts("Comment Threads:") : stat.to_s.titleize %></dt>
```

Comment:
Eegads, yes, that would be a hell of an if statement. It would probably be better to do them all individually, like the hits are:

```
      <dt class="hits"><%= ts("Hits:") %></dt>
      <dd class="hits"><%= @totals[:hits] %></dd>
```

...hmm. Actually, do you think I should do that?